src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/RadioButtonSamplePage.xaml

```diff
@@ -95,6 +95,39 @@
 			<sample:SamplePageLayout.CupertinoTemplate>
 				<DataTemplate>
 					<StackPanel>
+
```

Comment:
I'm changing the names for Cupertino_RadioButton[..] and adding Material_RadioButton to existing items